timestamp-tests-shared/src/main/java/com/palantir/timestamp/AllTimestampTests.java

```diff
@@ -0,0 +1,27 @@
+/**
+ * Copyright 2015 Palantir Technologies
+ *
+ * Licensed under the BSD-3 License (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://opensource.org/licenses/BSD-3-Clause
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.palantir.timestamp;
+
+import org.junit.runner.RunWith;
+import org.junit.runners.Suite;
+
+@RunWith(Suite.class)
+@Suite.SuiteClasses({
+        PersistentTimestampServiceTest.class,
+        RateLimitedTimestampServiceTest.class
```

Comment:
no need to do this if all the things labeled with `@Test` are going to be run (we do this for the other tests to get rid of some tests, and also to do some pre-init/post-cleanup)